### PR TITLE
25.0.0 RC2

### PR DESCRIPTION
**Pull request by blizzz**
* #34187
* #34213
* #34221
* #34236
* #34237
* #34251
* #34253
* #34257
* #34258
* #34268
* #34270
* #34285
* #34286
* #34293
* #34294
* #34295
* #34310
* #34315
* #34316
* #34317
* #34319
* #34320
* #34325
* nextcloud/activity#928
* nextcloud/activity#934
* nextcloud/activity#936
* nextcloud/firstrunwizard#745
* nextcloud/logreader#786
* nextcloud/logreader#790
* nextcloud/nextcloud_announcements#104
* nextcloud/nextcloud_announcements#107
* nextcloud/nextcloud_announcements#109
* nextcloud/notifications#1286
* nextcloud/notifications#1291
* nextcloud/notifications#1293
* nextcloud/photos#1286
* nextcloud/photos#1288
* nextcloud/photos#1299
* nextcloud/photos#1302
* nextcloud/related_resources#51
* nextcloud/related_resources#58
* nextcloud/serverinfo#402
* nextcloud/survey_client#145
* nextcloud/survey_client#148
* nextcloud/text#3058
* nextcloud/text#3060
* nextcloud/text#3065
* nextcloud/viewer#1387

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

**Review by nickvergessen — APPROVED** [review]

**Review by AndyScherzinger — APPROVED** [review]

**Comment by szaimen** [comment]
@blizzz are we allowed to merge backports during the RC phase?

File: version.php [hunk 1]
```diff
@@ -30,10 +30,10 @@
 // between betas, final and RCs. This is _not_ the public version number. Reset minor/patchlevel
 // when updating major/minor version number.
 
-$OC_Version = [25, 0, 0, 13];
+$OC_Version = [25, 0, 0, 14];
 
 // The human readable string
-$OC_VersionString = '25.0.0 RC 1';
+$OC_VersionString = '25.0.0 RC2';
```
[the remark this answers is not in the document]
Comment (blizzz) [reply version.php:36]: RC 1 was wrong

**Comment by blizzz** [comment]
> @blizzz are we allowed to merge backports during the RC phase?

It is not the last RC, so it's a bit less strict. But still: important bug fixes, no funny things, no features.

**Comment by andyxheli** [comment]
Hi all would we be able to add this in as well https://github.com/nextcloud/server/pull/32635

**Comment by blizzz** [comment]
> Hi all would we be able to add this in as well #32635

commented in the ticket. in doubt we'll have at least one more RC.

**Pull request merged**